src/components/NavBar.vue

```diff
@@ -20,4 +24,61 @@ import { RouterLink } from 'vue-router'
       <RouterLink to="#">Sponsors</RouterLink>
     </div>
   </div>
+
+  <!-- Mobile Navbar -->
+  <div class="lg:hidden fixed z-20 top-0 w-full flex justify-between items-center bg-slate py-3 drop-shadow-lg">
+    <RouterLink to="/" class="flex items-center">
+      <img class="-mr-1" src="../assets/logo.svg" alt="Biomod Logo" />
+      <span class="text-[#FFC000] font-bold text-xl">UBC Biomod</span>
+    </RouterLink>
+    <button class="mr-10" @click="sidebar = !sidebar">
+      <svg class="active:opacity-70 transition-all duration-200 ease-in-out" viewBox="0 0 10 8" width="35">
+        <path d="M1 1h8M1 4h 8M1 7h8" class="stroke-gold" stroke-width="1.0" stroke-linecap="round" />
+      </svg>
+    </button>
+  </div>
+
+  <Transition>
+    <template v-if="sidebar">
```

Comment:
Can you make the sidebar scrollable? Horizontal screen on mobile looks kinda wack, and you can't event access all the options.
<img width="541" alt="image" src="https://github.com/biomod2023/biomod2023/assets/62190721/d8b5d4e9-1050-44a3-8430-073ff3bc4ec5">

Reply:
Not sure if my end result is the optimal look but it'll do for now I think